static/app/components/loadingError.tsx

```diff
@@ -31,16 +29,18 @@ class LoadingError extends React.Component<Props> {
   render() {
     const {message, onRetry} = this.props;
     return (
-      <StyledAlert type="error">
-        <Content>
-          <IconInfo size="lg" />
-          <div data-test-id="loading-error-message">{message}</div>
-          {onRetry && (
+      <StyledAlert
+        type="error"
+        showIcon
+        trailingItems={
+          onRetry && (
             <Button onClick={onRetry} type="button" priority="default" size="small">
               {t('Retry')}
             </Button>
-          )}
-        </Content>
+          )
+        }
+      >
+        <div data-test-id="loading-error-message">{message}</div>
```

Comment:
instead of using a test-id, what do you think we only check for the text? `screen.getByText()`

Reply:
ahh good point! https://github.com/getsentry/sentry/pull/32781/commits/264dcfcfbdaab13762cb9fa9e1cba89d78b75fa9